src/Service.Host/client/src/actions/productsOnHoldReportActions.js

```diff
@@ -0,0 +1,31 @@
+import { CALL_API } from 'redux-api-middleware';
+import config from '../config';
+import * as actionTypes from './index';
+
+const fetchProductsOnHold = () => ({
+    [CALL_API]: {
+        endpoint: `${config.appRoot}/products/reports/products-on-hold`,
+        method: 'GET',
+        options: { requiresAuth: true },
+        headers: {
+            Accept: 'application/json'
+        },
+        types: [
+            {
+                type: actionTypes.REQUEST_PRODUCTS_ON_HOLD_REPORT,
+                payload: {}
+            },
+            {
+                type: actionTypes.RECEIVE_PRODUCTS_ON_HOLD_REPORT,
```

Comment:
I started using a potentially generalised version of these in the salesArticleCoreTypesReport. 
I haven't had a chance to pull it out into it's own reusable actions.
I wouldn't change this but might be worth bearing in mind next time you do a report.